scripts/prep.sh

```diff
@@ -88,6 +88,8 @@ export RUST_PACKAGE_NAME="ory-${PROJECT}-client"
 
 export NPM_NAME="@ory/$PROJECT-client"
 
+export ELIXIR_PACKAGE_NAME="ory_${PROJECT}_ex"
```

Comment:
Is that the nomenclautur for hex packages, that they end with `_ex`?

Reply:
It's so-so. I think we could drop the suffix and keep consistent on the ory package names 👍